src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -220,3 +220,66 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
         c.argument('vnet_name', help='Name of a virtual network.')
         c.argument('ip_address', help='IPv4 address or CIDR range.')
+
+    with self.argument_context('acr scope-map') as c:
+        c.argument('registry_name', options_list=['--registryName', '-r'], help='The name of the target registry.', required=True)
+
+    with self.argument_context('acr scope-map create') as c:
+        c.argument('description', options_list=['--description'], help='Description for the scope map. Max of 140 characters.', required=False)
+        c.argument('add_repository', nargs='+', options_list=['--add'], help='Repositories to be added. Add tuples (repository,action1,action2,...) separated by space.', required=True)
+        c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the target scope map.', required=True)
+
+    with self.argument_context('acr scope-map delete') as c:
+        c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the target scope map.', required=True)
+
+    with self.argument_context('acr scope-map update') as c:
+        c.argument('description', options_list=['--description'], help='Description for the scope map. Max of 140 characters.', required=False)
+        c.argument('add_repository', nargs='+', options_list=['--add'], help='Repositories to be added. Add tuples (repository,action1,action2,...) separated by space.', required=False)
+        c.argument('remove_repository', nargs='+', options_list=['--remove'], help='Repositories to be removed. Add tuples (repository,action1,action2,...) separated by space.', required=False)
+        c.argument('reset_map', options_list=['--reset'], help='Flag indicating whether scope map should be reset (erase all configurations)', action='store_true', required=False)
+        c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the target scope map.', required=True)
+
+    with self.argument_context('acr scope-map show') as c:
+        c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the target scope map.', required=True)
+
+    with self.argument_context('acr token') as c:
+        c.argument('registry_name', options_list=['--registryName', '-r'], help='The name of the target registry.', required=True)
+
+    with self.argument_context('acr token create') as c:
+        c.argument('token_name', options_list=['--name', '-n'], help='The name of the target token.', required=True)
```

Comment:
Same as above. You can add `token_name` to `acr token` group.

Reply:
Actually we also have the `az acr token list` command, which has only the registry name as an argument, so we can't have token name as a parameter for `acr token` group